lib/datadog_backup/resources/local_filesystem.rb

```diff
@@ -0,0 +1,51 @@
+# frozen_string_literal: true
+
+require 'fileutils'
+require 'json'
+require 'yaml'
+require 'deepsort'
+
+module DatadogBackup
+  class Resources
+    ##
+    # Meant to be mixed into DatadogBackup::Resources
+    # Relies on $options[:backup_dir] and $options[:output_format]
```

Comment:
I don't see `$options[:backup_dir]` mentioned in this file; is this comment accurate?